### PR TITLE
issues resolved 

### PR DESCRIPTION
**Pull request by Yash-Jain-1503**
block which appeared while opening the the pause as well as instructions button has been resolved.

### CODE REVIEW

File: constants.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 HIGHSCORE_FILE="highscore.txt"
 ARROW_IMAGE = "assets/arrow_1.png"
-BACKGROUND_IMAGE = "assets/background.png"
+BACKGROUND_IMAGE = "/home/yash/myprojects/Arcus-1/assets/90146416-vector-city-outdoor-day-landscape-house-and-street-buildings-outdoor-cityspace-disign-illustration-m.jpg"
```
Comment (sriyash421) [inline constants.py]: Change the path of the image. Else it will run only on your local system

**Comment by Yash-Jain-1503** [comment]
What is the problem with my PR kindly tell 
me its showing new background but I have 
not changed the background I have just removed the blocks in the background when we click pause and ins. button rest all things are same

**Review by sriyash421 — COMMENTED** [review]
> What is the problem with my PR kindly tell
> me its showing new background but I have
> not changed the background I have just removed the blocks in the background when we click pause and ins. button rest all things are same

I have mentioned the change required in the pr.

**Comment by Yash-Jain-1503** [comment]
changed the path of the image

**Comment by sriyash421** [comment]
Please resolve the merge conflicts and send. a new pr. I am closing this one.
I know this might be frustrating but it will help you send proper PR in future open source projects.

**Comment by Yash-Jain-1503** [comment]
thanks for guiding me now I have resolved all issue regarding merge as well
as blocks and sent you a new PR.

**Pull request closed without merging**